dallinger/command_line.py

```diff
@@ -806,21 +812,22 @@ def export(app, local):
             " --app " + app_name(id),
             shell=True)
 
-        dump_path = dump_database(id)
+    try:
+        subprocess.call([
+            "dropdb",
+            app_name(id),
+        ])
+    except Exception:
+        pass
 
-        try:
-            subprocess.check_call([
-                "pg_restore",
-                "--verbose",
-                "--no-owner",
-                "--clean",
-                "-d",
-                "dallinger",
-                os.path.join("data", id, "data.dump")
-            ])
-
-        except Exception:
-            pass
+    subprocess.call([
```

Comment:
Thanks, that makes sense. We have control over the Postgres version used on Heroku, so maybe we should expose that choice as a configuration variable and then check that when the data is exported, the local Postgres version matches the version of the database being exported.